assets/i18n/strings.i18n.json

```diff
@@ -283,7 +283,7 @@
     "status_failure_timeout_description": "The installation took too long to finish.\n\nWould you like to try again?",
     "status_failure_storage_description": "The installation failed due to insufficient storage.\n\nFree up some space and try again.",
     "status_failure_invalid_description": "The installation failed due to the patched app being invalid.\n\nUninstall the app and try again?",
-    "status_failure_incompatible_description": "The app is incompatible with this device.\n\nContact the developer of the app and ask for support.",
+    "status_failure_incompatible_description": "The app is incompatible with this device.\n\nEnsure that the APK you are using is supported by this device and try again.",
```

Comment:
The APK is incompatible, ensuring it is compatible or retrying when it is incompatible doesn't make sense

Reply:
You just changed "you are using" to "you use"

Reply:
"You use" refers to the future, meaning that the user should use an APK that has been ensured to be supported.

It can instead say something such as, 1. `Try again with an APK that is supported by this device.` or 2., `Use an APK that is supported by this device and try again.`

Reply:
I deleted it because I thought it had the same issue as the current string 🫠

`Use an APK that is supported by this device and try again.`